README.md

```diff
@@ -2,16 +2,29 @@
 
 ImageLab is a framework that allows students to develop
 image modification processors (filters) and to experience
-the results visually and aurally.
+the results visually and aurally. This project can be ran as a BlueJ Project, directly from the command line, or built with a gradle wrapper. 
 
-## Contents
+Project Website & User Instructions: https://metrocs.github.io/imagelab/
+_____
+
+## Table of Contents
++ [Repository Contents](#repository-contents)
++ [Build Instructions](#build-instructions)
++ [License](#license)
+
+## Repository Contents 
 - The __imagelab__ and __sound__ packages may be provided in an `imagelab.jar` file.
```

Comment:
This is a historical perspective. Those packages are now provided in an imagelab.jar file. Please change "may be" to "are".